hawaii-logging/src/main/java/org/hawaiiframework/logging/web/util/GraphQlTransactionTypeSupplier.java

```diff
@@ -36,18 +36,27 @@
 public class GraphQlTransactionTypeSupplier implements TransactionTypeSupplier {
 
     private static final String GRAPHQL_URL_IDENTIFIER = "graphql";
+    private static final String MULTI_PART_MIME_TYPE = "multipart/form-data";
 
     private static String getPostBody(final HttpServletRequest servletRequest) throws IOException {
         return IOUtils.toString(servletRequest.getInputStream(),
             servletRequest.getCharacterEncoding());
     }
 
+    @SuppressWarnings("checkstyle:ReturnCount")
     @Override
     public String getTransactionType(final ResettableHttpServletRequest request) throws IOException {
-        if (!request.getServletPath().contains(GRAPHQL_URL_IDENTIFIER)) {
+        if (request.getServletPath().contains(GRAPHQL_URL_IDENTIFIER)) {
             return null;
         }
         try {
+            // Collect the graphQl multiPart request with.
+            // request.getParts().stream().findFirst().get().getContentType()
+            String contentType = request.getContentType();
+            if (contentType != null && contentType.contains(MULTI_PART_MIME_TYPE)) {
+                return "Graphql.multiPart";
```

Comment:
See media type voter to convert this into the media type.
Check against the known media type as well.